shared/chat/conversation/input-area/filepicker-popup/index.types.js.flow

```diff
@@ -5,5 +5,5 @@ export type Props = {
   attachTo: ?React.Component<any, any>,
   visible: boolean,
   onHidden: () => void,
-  onSelect: string => void,
+  onSelect: (mediaType: string, location: string) => void,
```

Comment:
actually should be 
mediaType: 'photo' | 'video' | 'mixed', location: 'library' | 'camera'